src/chrome/content/rules/Trakehner-nord-west.de.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="Trakehner-nord-west.de">
+	<target host="www.trakehner-nord-west.de" />
+	<target host="trakehner-nord-west.de" />
+
+	<securecookie host="^(www\.)?trakehner-nord-west\.de$" name=".*" />
+
+	<rule from="^http://(www\.)?trakehner-nord-west\.de/" to="https://$1trakehner-nord-west.de/" />
```

Comment:
Replace this line by

		<rule from="^http://(?:www\.)?trakehner-nord-west\.de/" to="https://www.trakehner-nord-west.de/" />

because `https://trakehner-nord-west.de/` contains the following base line in the `<HEAD>` section:

	<base href="https://www.trakehner-nord-west.de/" />

This avoids a hostname change when navigating to any other page of that site.